### PR TITLE
Added display of current Apple News publish state to admin screens

### PR DESCRIPTION
**Pull request by bcampeau**

### CODE REVIEW

File: admin/class-admin-apple-meta-boxes.php [hunk 1]
```diff
@@ -165,6 +168,7 @@ public function publish_meta_box( $post ) {
 			<br/><?php esc_html_e( 'Modified at', 'apple-news' ) ?>: <?php echo esc_html( $modified_at ) ?>
 			<br/><?php esc_html_e( 'Share URL', 'apple-news' ) ?>: <a href="<?php echo esc_url( $share_url ) ?>" target="_blank"><?php echo esc_html( $share_url ) ?></a>
 			<br/><?php esc_html_e( 'Revision', 'apple-news' ) ?>: <?php echo esc_html( get_post_meta( $post->ID, 'apple_news_api_revision', true ) ) ?>
+			<br/><?php esc_html_e( 'State', 'apple-news' ) ?>: <?php echo esc_html( $state ) ?>
```
Comment (joshkadis) [inline admin/class-admin-apple-meta-boxes.php]: @mboynes says that line breaks are for addresses and poetry

File: admin/class-admin-apple-news-list-table.php [hunk 2]
```diff
@@ -219,10 +236,11 @@ public function column_title( $item ) {
...
-			'title'      => __( 'Title', 'apple-news' ),
-			'updated_at' => __( 'Last updated at', 'apple-news' ),
-			'sync'       => __( 'Apple News Status', 'apple-news' ),
+			'cb'        	=> '<input type="checkbox">',
+			'title'     	=> __( 'Title', 'apple-news' ),
+			'updated_at' 	=> __( 'Last updated at', 'apple-news' ),
+			'status' 			=> __( 'Apple News Status', 'apple-news' ),
+			'sync'       	=> __( 'Sync Status', 'apple-news' ),
 		) );
 	}
```
Comment (joshkadis) [inline admin/class-admin-apple-news-list-table.php]: indentation got wonky here

**Comment by joshkadis** [comment]
:bird:

File: admin/class-admin-apple-meta-boxes.php (hunk 1 above)
Comment (bcampeau) [inline admin/class-admin-apple-meta-boxes.php]: In this case, it makes sense for display purposes.

Comment (joshkadis) [inline admin/class-admin-apple-meta-boxes.php]: That's what I keep telling him.

**Pull request merged**